lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-ansible_tower-configuration_manager-playbook.rb

```diff
@@ -0,0 +1,5 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_AnsibleTower_ConfigurationManager_Playbook < MiqAeServiceConfigurationScriptPayload
+    expose :run
```

Comment:
Why are you exposing `run` in this PR?  The method doesn't even apply to this subclass.